pkg/util/overridemanager/imageoverridemanager.go

```diff
@@ -0,0 +1,141 @@
+package overridemanager
```

Comment:
I'd prefer to rename the file to `imageoverrider.go`.

Reply:
done.